tests/e2e/simple-app.test.ts

```diff
@@ -12,9 +12,12 @@ test('Renders the Home page correctly', async ({ page, simple }) => {
 
   await expect(page).toHaveTitle('Simple Next App')
 
-  expect(headers['cache-status']).toBe(
-    '"Next.js"; hit\n"Netlify Durable"; fwd=miss\n"Netlify Edge"; fwd=miss',
-  )
+  expect(headers['cache-status']).toMatch(/^"Next.js"; hit$/m)
+  expect(headers['cache-status']).toMatch(/^"Netlify Edge"; fwd=miss$/m)
```

Comment:
This is alternative to previously merged updated assertions ( https://github.com/netlify/next-runtime/pull/2527 ) that does kind of preserve previous assertions and skips checking on "Netlify Durable" cache status